src/main/kotlin/com/example/passmanager/configuration/ValidIdFormat.kt

```diff
@@ -0,0 +1,6 @@
+package com.example.passmanager.configuration
+
+@Target(AnnotationTarget.CLASS)
+@Retention(AnnotationRetention.RUNTIME)
+annotation class ValidIdFormat
```

Comment:
```suggestion
annotation class ValidObjectIdFormat
```
Just to have more clear name